homework.cls

```diff
@@ -104,12 +123,12 @@
 % Specify how the title is printed
 \renewcommand\maketitle
 {
-  \thispagestyle{firstpage}
-  \begingroup
-  \center
-  \sffamily\bfseries\Large\@hwtitle
-  \endcenter
-  \endgroup
+  \thispagestyle{plain}
+  \begin{center}
+    \textsf{\textbf{\Large\@hwtitle}}
+  \end{center}
+  \textbf{Name:} \@name \ (\@studentid)\hspace{\fill} \textbf{Due:} \@duedate \\
+  \textbf{Email:} \href{mailto:\@email}{\texttt{\@email}} \hspace{\fill} \textbf{Course:} \@courseid
```

Comment:
Does this change make the name appear on the first page twice? Once in the upper right header and once below the title? I can't build an example doc because of unrelated errors, so I can't tell